mandatory/3-playing-computer.js

```diff
@@ -7,12 +7,35 @@
   Answer the following questions:
 
   1. This program throws an error. Why? (If you can't find it, try executing it).
+     
+      The error is at console.log(b), this is because 'b' is not defined and the error message will log out 'undefined'.
+ 
   2. Remove the line that throws the error.
   3. What is printed to the console?
+      
+      2;
+      6;
+      4;
+      6;
```

Comment:
Uh-oh, something has gone wrong starting from line 19. Try going through this again to see what might have gone wrong.

Reply:
@louisechow : 
These are my new values:

 ```
     2;
      6;
      4;
      9;
      6;
      12;
      8; 
```

Reply:
The second-to-last value is not correct.

Reply:
@louisechow , you totally right,,,,since i = 3 and a = 10, the new vlaue should be 13. Thanks again for noticing this mistake